.github/workflows/ci.yaml

```diff
@@ -81,7 +81,7 @@ jobs:
     name: Go lint
     runs-on: ubuntu-latest
     env:
-      GOLANGCI_LINT_VERSION: v1.63.4
+      GOLANGCI_LINT_VERSION: v1.64.6
```

Comment:
We were on 1.64.4 elsewhere so we probably need to bump everywhere.

Reply:
Luckily I don't think the new versions make that big of a difference (at least between 1.63 and 1.64). The main thing for some of them recently was getting in Go 1.24 support. But yeah, we should start rolling all the versions forward as we get to them.